<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationPanel</class>
 <widget class="QWidget" name="InformationPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="layout_info">
        <item>
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_forename">
          <property name="text">
           <string>Forename: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_logout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_date">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_logout">
          <property name="styleSheet">
           <string notr="true">QPushButton
{
    padding: 10px;
    border:none;
    font: 16pt 'Open Sans';
    color:rgb(252, 255, 247);
    background-color: rgb(33, 160, 160);
}

QPushButton:hover
{
    background-color: rgba(33, 160, 160,200);
}

QPushButton:pressed
{
    background-color: rgb(4, 95, 92);
}
</string>
          </property>
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
